server.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var curses = require('./curses.json').curses;
+
+var app = express();
+var port = process.env.PORT || 3000;
+
+app.use(express.static(__dirname + '/'));
```

Comment:
Could you update this to use the `path` library? CodeClimate flagged with with `Use path.join() or path.resolve() instead of + to create paths. `. re: http://eslint.org/docs/rules/